dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -306,13 +308,20 @@ internal constructor(
          * UI interaction events (e.g.: taps, scrolls, swipes) and automatically send those as RUM UserActions for you.
          * @param touchTargetExtraAttributesProviders an array with your own implementation of the
          * target attributes provider.
+         * @param interactionPredicate an interface to provide custom values for the
```

Comment:
it seems description should be updated: it doesn't provide custom values for now, just different name. Or it is written in assumption or future extensibility of `InteractionPredicate`?

Reply:
It is with the assumption of future extensibility